eng/config/rulesets/Shipping.ruleset

```diff
@@ -154,8 +154,6 @@
     <Rule Id="xUnit2013" Action="None" /> <!-- "do not use Assert.Equal() to check for collection size" is a valid assert, but very noisy right now -->
     <Rule Id="xUnit2015" Action="None" /> <!-- "do not use typeof() expression to check the exception type. " is a valid assert, but very noisy right now -->
     <Rule Id="xUnit2017" Action="None" /> <!-- "do not use Contains() to check if a value exists in a collection" is a valid assert, but very noisy right now -->
-    <Rule Id="xUnit2018" Action="None" /> <!-- "do not compare an object's exact type to the abstract class" is a valid assert, but very noisy right now -->
-collection.
```

Comment:
Why is this being removed?

Reply:
Nevermind

Reply:
Just moved. I moved it into the editorconfig file so I could suppress it using editorconfig and verify things were working.